src/Components/Cart.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { deleteCartItemThunk, deleteGuestItem } from '../redux/store';
+
+class _Cart extends Component {
+  render() {
+    const { cart, guestCart, destroy, deleteGuestItem, orders, auth, products } = this.props;
+    let userCart, ownCart, cartTotal;
+    if(auth.name !== 'Guest'){
+      const orderAuth = orders.filter(order => order.userId === auth.id);
+      userCart =
+      orderAuth.length !== 0
+        ? cart.filter(item => item.orderId === orderAuth[0].id)
```

Comment:
In what case would a `LineItem` be associated with an order that isnt the right order? This check seems unnecessary or something very strange is happening with your reduction of the response from the back end.